src/unity/python/README.md

```diff
@@ -1,132 +0,0 @@
-* **Visual:** Built-in, streaming visualizations to explore your data
-* **Flexible:** Supports text, images, audio, video and sensor data
-* **Fast and Scalable:** Work with large datasets on a single machine
-* **Ready To Deploy:** Export models to Core ML for use in iOS, macOS, watchOS, and tvOS apps
-
-Example: Image classifier with a few lines of code
---------------------------------------------------
-
-If you want your app to recognize specific objects in images, you can build your own model with just a few lines of code:
-
-```python
-import turicreate as tc
-
-# Load data 
-data = tc.SFrame('photoLabel.sframe')
-
-# Create a model
-model = tc.image_classifier.create(data, target='photoLabel')
-
-# Make predictions
-predictions = model.predict(data)
-
-# Export to Core ML
-model.export_coreml('MyClassifier.mlmodel')
-```
- 
-It's easy to use the resulting model in an [iOS application](https://developer.apple.com/documentation/vision/classifying_images_with_vision_and_core_ml):
-
-<p align="center"><img src="https://docs-assets.developer.apple.com/published/a2c37bce1f/689f61a6-1087-4112-99d9-bbfb326e3138.png" alt="Turi Create" width="600"></p>
-
-With Turi Create, you can can tackle a number of common scenarios:
-* [Recommender systems](https://apple.github.io/turicreate/docs/userguide/recommender/introduction.html)
-* [Image classification](https://apple.github.io/turicreate/docs/userguide/image_classifier/introduction.html)
-* [Image similarity](https://apple.github.io/turicreate/docs/userguide/image_similarity/introduction.html)
-* [Object detection](https://apple.github.io/turicreate/docs/userguide/object_detection/introduction.html)
-* [Activity classifier](https://apple.github.io/turicreate/docs/userguide/activity_classifier/introduction.html)
-* [Text classifier](https://apple.github.io/turicreate/docs/userguide/text_classifier/introduction.html)
-
-You can also work with essential machine learning models, organized into algorithm-based toolkits:
-* [Classifiers](https://apple.github.io/turicreate/docs/userguide/supervised-learning/classifier.html)
-* [Regression](https://apple.github.io/turicreate/docs/userguide/supervised-learning/regression.html)
-* [Graph analytics](https://apple.github.io/turicreate/docs/userguide/graph_analytics/intro.html)
-* [Clustering](https://apple.github.io/turicreate/docs/userguide/clustering/intro.html)
-* [Nearest Neighbors](https://apple.github.io/turicreate/docs/userguide/nearest_neighbors/nearest_neighbors.html)
-* [Topic models](https://apple.github.io/turicreate/docs/userguide/text/intro.html)
-
-System Requirements
--------------------
-
-* Python 2.7 (Python 3.5+ support coming soon)
-* x86\_64 architecture
-* macOS 10.11+, Linux with glibc 2.12+ (including WSL on Windows 10)
-
-Installation
-------------
-
-For detailed instructions for different varieties of Linux see [LINUX\_INSTALL.md](https://github.com/apple/turicreate/LINUX_INSTALL.md).
```

Comment:
This section is not required in the pypi readme.

Reply:
I think system requirements are actually relevant, aren't they?